yaksh/forms.py

```diff
@@ -196,7 +199,7 @@ class ProfileForm(forms.ModelForm):
     class Meta:
         model = Profile
         fields = ['first_name', 'last_name', 'institute',
-                'department', 'roll_number', 'position']
+                'department', 'roll_number', 'position', 'timezone']
```

Comment:
Nothing wrong here, just a suggestion, use ```exclude = ['fieldname']``` so that you don't need to add a field everytime you make some change.